compiler/rustc_middle/src/middle/cstore.rs

```diff
@@ -200,7 +200,7 @@ pub trait CrateStore {
     // "queries" used in resolve that aren't tracked for incremental compilation
     fn crate_name_untracked(&self, cnum: CrateNum) -> Symbol;
     fn crate_is_private_dep_untracked(&self, cnum: CrateNum) -> bool;
-    fn crate_disambiguator_untracked(&self, cnum: CrateNum) -> CrateDisambiguator;
+    fn stable_crate_id_untracked(&self, cnum: CrateNum) -> StableCrateId;
```

Comment:
The StableCrateId is supposed to be stable for incr. comp., so the `untracked` name is not required.

Reply:
Both `stable_crate_id_untracked` and `crate_name_untracked` should be renamed, right? Changing the crate name will change the `StableCrateId` too.

Reply:
I agree. As a consequence, `crate_name` may not even need to be a query. Let's leave this rabbit hole to a follow-up query.

Reply:
cc https://github.com/rust-lang/rust/pull/85837